### ISSUE TITLE
Android 11  - No valid challenge was passed ...

### ISSUE THREAD

**Issue by Arrow433** [PhilippC/keepass2android#1571]
After switching from Android 8 to Android 11 with version 1.08d-r5 I can no longer open my database (generated with Kepass 2.47) with password + challenge response (with yubikey). ykDroid is installed. I get the message: No valid challenge was passed ... On Android 8 everything works as before.

**Comment by Arrow433** [comment]
Keepass2Android can no longer open the XML help file for the Yubikey usage and therefore returns a wrong challenge. The error probably comes from the fact that access to all files is no longer automatically allowed in android 11. The feature is called scoped storage. Keepass2android is only allowed to access media content and not all file content in Android 11. Enabling the app to access all files is allowed by Google and probably has to be requested by the developer. After a check, it will be unlocked.

**Comment by dprosperino** [comment]
Does it mean that, at least for now, it is not possible to use the Challange Response Feature under Android 11?

(I have the same issue with my device running Android 10; when I try to specify the OTP auxiliary file, it says "No valid challenge submitted")

**Comment by Arrow433** [comment]
Yes, I can no longer log in withe Challange Response Feature since the change to Android 11.
